nixos/modules/virtualisation/docker.nix

```
@@ -160,6 +160,34 @@ in
           which the prune will occur.
         '';
       };
+
+      randomizedDelaySec = mkOption {
+        default = "0";
+        type = types.singleLineStr;
+        example = "45min";
+        description = lib.mdDoc ''
+          Add a randomized delay before each auto prune.
+          The delay will be chosen between zero and this value.
+          This value must be a time span in the format specified by
+          {manpage}`systemd.time(7)`
+        '';
+      };
+
+      persistent = mkOption {
+        default = true;
+        type = types.bool;
+        example = false;
+        description = lib.mdDoc ''
```

```suggestion
        description = ''
```
These are no longer needed.